crates/rustc_codegen_spirv/src/codegen_cx/mod.rs

```diff
@@ -277,6 +277,14 @@ pub struct CodegenArgs {
     pub disassemble_fn: Option<String>,
     pub disassemble_entry: Option<String>,
     pub disassemble_globals: bool,
+
+    // spirv-val flags
+    pub relax_struct_store: bool,
+    pub relax_logical_pointer: bool,
+    pub relax_block_layout: Option<bool>,
```

Comment:
So this is basically a tri-state? It stands out a bit here, maybe deserves a comment (although this is kind of a deep location).

Reply:
Yeah, it's... a little weird? Even weirder is that I don't think the CLI of `spirv-val` exposes this tri-state, it's only `Some(true)` and `None`. I thiiiink the tri-state is:

- `Some(true)` -> force relax-block-layout on
- `Some(false)` -> force relax-block-layout off
- `None` -> relax-block-layout is on if vulkan version >=1.1, off if ==1.0